wlp-managed-8.5/src/main/java/org/jboss/arquillian/container/was/wlp_managed_8_5/WLPManagedContainer.java

```diff
@@ -576,10 +707,23 @@ private Element createApplication(Document doc, String deploymentName, String ar
          application.appendChild(sharedLib);
       }
 
+      if(containerConfiguration.getSecurityConfiguration() != null) {
```

Comment:
Sure, where shall I write the description? I guess I cannot edit the link you refered to